crypto/ec/ecdsa_ossl.c

```diff
@@ -99,14 +105,17 @@ static int ecdsa_sign_setup(EC_KEY *eckey, BN_CTX *ctx_in,
 
         /*
          * We do not want timing information to leak the length of k, so we
-         * compute G*k using an equivalent scalar of fixed bit-length.
+         * compute G^k using an equivalent scalar of fixed bit-length.
```

Comment:
Well, but in EC case `G*k` is more appropriate, because we are referring to point *multiplication* :-) Well, it's nothing like normal multiplication, but nevertheless it's referred to as one. Maybe one can use `G×k` to eliminate ambiguity...

Reply:
Now I know where that `*` came from.  You are correct of course and I've changed this back.